southpark-incremental-indexing/tests/test_southparksearch.py

```diff
@@ -0,0 +1,79 @@
+__copyright__ = "Copyright (c) 2020 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+import json
+import os
+import sys
+import subprocess
+
+from jina.flow import Flow
+import pytest
+
+NUM_DOCS = 100
+TOP_K = 3
+INDEX_FLOW_FILE_PATH = 'flow-index.yml'
+QUERY_FLOW_FILE_PATH = 'flow-query.yml'
+
+
+def config(tmpdir):
+    os.environ['JINA_DATA_FILE_1'] = 'tests/data-index-1.csv'
+    os.environ['JINA_DATA_FILE_2'] = 'tests/data-index-2.csv'
+    os.environ['JINA_WORKSPACE'] = str(tmpdir)
+    os.environ['JINA_PORT'] = str(45678)
+
+
+def index_documents():
+    f = Flow().load_config(INDEX_FLOW_FILE_PATH)
+
+    with f:
+        f.index_lines(filepath=os.environ['JINA_DATA_FILE_1'],
+                      batch_size=8,
+                      size=NUM_DOCS)
+
+    # close flow and index new set of docs
+    with f:
+        f.index_lines(filepath=os.environ['JINA_DATA_FILE_2'],
+                      batch_size=8,
+                      size=NUM_DOCS)
+
+
+def call_api(url, payload=None, headers={'Content-Type': 'application/json'}):
+    import requests
+    return requests.post(url, data=json.dumps(payload), headers=headers).json()
+
+
+def get_results(query, top_k=TOP_K):
+    return call_api(
+        'http://0.0.0.0:45678/api/search',
+        payload={"top_k": top_k, "mode": "search", "data": [f"text:{query}"]}
+    )
+
+
+def get_flow():
+    f = Flow().load_config(QUERY_FLOW_FILE_PATH)
+    f.use_rest_gateway()
+    return f
+
+
+@pytest.fixture
+def queries():
+    return [("Don't say anything\n",
+             ["Don't say anything\n", "Check that: I'll watch that game.\n", "I'm trying to talk\n"]),
```

Comment:
Do the results come from different sources?

Reply:
Yep